app/src/styles/_uswds-theme.scss

```diff
@@ -19,6 +19,9 @@ in the form $setting: value,
   $utilities-use-important: true,
 
   // Prettier defaults for typography
```

Comment:
question: what does this comment mean? anything to do with the "prettier" tool?

Reply:
```suggestion
  // Use USWDS defaults for typography
```

Reply:
@lorenyu Just updated this using Andy's suggestion. This was previously here for context on why the default sans-serif font family was being overridden.